roles/partition-disk/tasks/remove_home_partition.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: "Unmount /home partition"
+    mount:
+      name: /home
+      state: unmounted
+  - name: "Remove /home entry from /etc/fstab"
+    mount:
+      name: /home
+      state: absent
+  - name: "Remove /home Logical Volume"
+    when: item.key in "lv_home"
+    lvol:
+      vg: "{{ item.value.vg }}"
+      lv: "{{ item.key }}"
+      state: absent
+      force: yes
+    with_dict: "{{ ansible_lvm.lvs }}"
+...
```

Comment:
I think these can be dropped

Reply:
I use that for YAML files to mark end of the file, as I had read about it here, https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics  :)

Reply:
I learned something new today. I have never seen that before so I might recommend not including them to reduce confusion. I do a see a mixed use of `---` at the start of YAML across all YAML I have run across.